homeassistant/components/vicare/__init__.py

```diff
@@ -113,8 +121,36 @@ def get_supported_devices(
     devices: list[PyViCareDeviceConfig],
 ) -> list[PyViCareDeviceConfig]:
     """Remove unsupported devices from the list."""
+
+    _LOGGER.debug("Ignoring unsupported devices (%s)", UNSUPPORTED_DEVICES)
     return [
         device_config
         for device_config in devices
         if device_config.getModel() not in UNSUPPORTED_DEVICES
     ]
+
+
+def get_online_devices(
+    hass: HomeAssistant,
+    devices: list[PyViCareDeviceConfig],
+) -> list[PyViCareDeviceConfig]:
+    """Remove offline devices from the list."""
+
+    _LOGGER.debug("Ignoring offline devices")
+    result = []
+    for device_config in devices:
+        if device_config.isOnline():
+            result.append(device_config)
+        else:
+            ir.create_issue(
```

Comment:
We should not raise repair issues for device being offline.
Instead, make sure the entities are reporting in their availability correctly (reflected by their `unavailable` state).

Reply:
removed